docs/api/v1.md

```diff
@@ -0,0 +1,117 @@
+# Spice API
+## Version `v1`
+
+All routes are prefixed with `/api/v1/`
```

Comment:
I agree it's unnecessary, but it doesn't hurt. It may be useful for feature flipping and refactoring though (and does help support the option of multiple frontend clients, even though that's not part of the design).